internal/kubeconfig/serviceaccount.go

```diff
@@ -28,15 +28,16 @@ func getServiceAccountToken(ctx context.Context, c clientset.Interface, name str
 		log.Printf("Service Account not created: %v", err)
 	}
 
-	// Get token
-	_, token, err = getReferencedServiceAccountToken(c.(*clientset.Clientset), ns, name, true)
+	// check for errors
+	_, _, err = getReferencedServiceAccountToken(c.(*clientset.Clientset), ns, name, true)
 	if err != nil {
 		log.Fatal(err)
 	}
 
```

Comment:
looks like you are not returning the token anymore, is this intentional?

Reply:
the comment branch is outdated I think, i see the token in my version in an explicit way

Reply:
right, my fault